api/src/main/resources/jakarta/persistence/orm_3_2.xsd

```diff
@@ -387,7 +387,7 @@
       <xsd:element name="basic" type="orm:basic"
                    minOccurs="0" maxOccurs="unbounded"/>
       <xsd:element name="version" type="orm:version"
-                   minOccurs="0" maxOccurs="unbounded"/>
+                   minOccurs="0"/>
```

Comment:
add this change also to ch12, around line 1349, please

Reply:
I found one more thing to update (unrelated to this), so I'll fix this in the spec doc with that change